hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/web/DatanodeHttpServer.java

```diff
@@ -144,7 +147,16 @@ public DatanodeHttpServer(final Configuration conf,
     confForCreate.set(FsPermission.UMASK_LABEL, "000");
 
     this.bossGroup = new NioEventLoopGroup();
-    this.workerGroup = new NioEventLoopGroup();
+    int workerCount = conf.getInt(DFS_DATANODE_NETTY_WORKER_NUM_THREADS_KEY,
+        DFS_DATANODE_NETTY_WORKER_NUM_THREADS_DEFAULT);
+    if (workerCount < 0) {
+      LOG.warn("The value of " +
+          DFS_DATANODE_NETTY_WORKER_NUM_THREADS_KEY + " is less than 0, will use default value: " +
+          DFS_DATANODE_NETTY_WORKER_NUM_THREADS_DEFAULT);
+      workerCount = DFS_DATANODE_NETTY_WORKER_NUM_THREADS_DEFAULT;
```

Comment:
you logger format {} instead of concat

Reply:
@ayushtkn Thanks for your review. I have revised it. could you take a look? Thanks.